### ISSUE TITLE
Cannot find module when trying to run on Windows

### ISSUE THREAD

**Issue by xiphirx** [facebook/flipper#189]
I'm trying to run Sonar on Windows, which I'm not entirely sure if we're trying to support, but I've seen some commits recently that indicate that we might be adding in support. I'm running Windows 10 + cygwin but I get a weird module not found error:
```
~/dev/Sonar$ yarn
yarn install v1.5.1
(node:13788) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.4: The platform "win32" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
info 7zip-bin-mac@1.0.1: The platform "win32" is incompatible with this module.
info "7zip-bin-mac@1.0.1" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > eslint-config-fbjs@2.0.1" has incorrect peer dependency "babel-eslint@^7.2.3".
[4/4] Building fresh packages...
$ node node_modules/electron/install.js && node scripts/yarn-install.js
📦  Installed all dependencies!
Done in 62.96s.

 ~/dev/Sonar$ yarn start
yarn run v1.5.1
(node:4104) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
$ cross-env NODE_ENV=development node scripts/start-dev-server.js
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module './_baseHas'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (F:\cygwin\home\xiphi\dev\Sonar\static\node_modules\lodash\has.js:1:77)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
error An unexpected error occurred: "Command failed.
Exit code: 1
Command: F:\\cygwin\\bin\\bash.exe
Arguments: -c cross-env NODE_ENV=development node scripts/start-dev-server.js
Directory: F:\\cygwin\\home\\xiphi\\dev\\Sonar
Output:
".
info If you think this is a bug, please open a bug report with the information provided in "F:\\cygwin\\home\\xiphi\\dev\\Sonar\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

Any idea on what could be going wrong here?

**Comment by passy** [comment]
I don't think we've got anyone on the team with Windows knowledge, so the build support for that would likely have to come from the community (in the broader sense of outside the team at least).

**Comment by xiphirx** [comment]
Looked into this a bit more because, again, I wanted to use Flipper on my home PC: It seems like yarn or npm is unable to properly install lodash on Windows... or at least for my setup. I had to grab the lodash package from https://www.jsdelivr.com/package/npm/lodash, untar it and move the contents into `static/node_modules/lodash`. Afterwards, Flipper opens right up!

Closing because this isn't an issue with Flipper. Yarn seems to be a bit iffy around this, I might raise an issue there instead.
